ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -48,114 +30,151 @@ let update_mh_info interface =
   in
   ()
 
-let _restart_stunnel_no_cache ~__context ~accept =
-  let (_ : Thread.t) =
-    Thread.create (fun () -> Helpers.Stunnel.restart ~__context ~accept) ()
-  in
-  ()
+module Stunnel = struct
```

Comment:
it's nice to add a sig so that you can be sure nobody is using `stunnel_accept_m` outside this module